README.md

```diff
@@ -3,9 +3,7 @@ This is a Chrome extension (written in C#) that shows a Xamarin.Forms page insid
 ## How to build
 1. Open the App8.sln file with Visual Studio 2019 or VS for Mac 8.6.7 (or newer)
 1. Publish the .wasm project to a local folder
-1. Copy all files from browserExtensionAssets/ folder into it.
-1. Delete subfolders which start with underscore.
-1. Edit index.html to include `<link rel="stylesheet" type="text/css" href="./stylesheet.css" />`.
+1. Run `dotnet fsi "scripts\post-build.fsx" <path-to-browserExtensionAssets-folder> <path-to-publish-result-folder>`
```

Comment:
to even simplify this more, let's create a `make.bat` file that calls the above, and chooses the folders for the user (and so in the instructions we say "your extension will be generated in the `bin` subfolder" for example)

Reply:
That is not in scope of this PR

Reply:
Changing the README file to simplify instructions is a side-effect of simplifying the generation of the extension folder. Simplifying the build instructions even more is just a better way to implement this side-effect.